src/server/pkg/storage/chunk/storage.go

```diff
@@ -6,17 +6,21 @@ import (
 	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
 )
 
+const (
+	// Prefix is the chunk storage prefix.
+	Prefix = "chunks"
```

Comment:
Does prefix need to be public? I was kind of expecting it would be a private detail of this package.

Reply:
Nah, the fileset prefix does not need to be exported either. It was needed at one point, but it is probably better if clients did not depend on this.